app/generated/Ward-svcServiceClientPb.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * @fileoverview gRPC-Web generated client stub for 
+ * @enhanceable
+ * @public
+ */
+
+// Code generated by protoc-gen-grpc-web. DO NOT EDIT.
+// versions:
+// 	protoc-gen-grpc-web v1.4.2
+// 	protoc              v3.20.3
+// source: ward-svc.proto
+
+
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
this doesn't sound confidence inspiring
the generated code should at least be void of type errors

Reply:
Disabling the typechecking does not imply errors inside the generated code.

One could even argue that generated source code is by definition (as you said) syntactically error-free and therefore does not need to be tested. You are welcome to choose how deep we should go in this discussion, or if we should dwell on it.

Reply:
disabling error checking affects the behaviour

generated code is definitely not always error free and using `@ts-nocheck` to circumvent checks implies that it is indeed not free of errors as it wouldn't be needed otherwise

Reply:
From a functional point of view, we should be satisfied with the codegen. If we do not like the code style, we should address this in a new issue after the merge.

Reply:
https://github.com/helpwave/web/issues/89